pkg/minikube/registry/drvs/hyperv/hyperv.go

```diff
@@ -91,13 +91,19 @@ func status() registry.State {
 	ctx, cancel := context.WithTimeout(context.Background(), 8*time.Second)
 	defer cancel()
 
-	cmd := exec.CommandContext(ctx, path, "-NoProfile", "-NonInteractive", "@(Get-Wmiobject Win32_ComputerSystem).HypervisorPresent")
+	cmd := exec.CommandContext(ctx, path, "-NoProfile", "-NonInteractive", "@(Get-CimInstance Win32_ComputerSystem).HypervisorPresent")
 	out, err := cmd.CombinedOutput()
 
 	if err != nil {
-		errorMessage := fmt.Errorf("%s failed:\n%s", strings.Join(cmd.Args, " "), out)
-		fixMessage := "Start PowerShell as an Administrator"
-		return registry.State{Installed: false, Running: true, Error: errorMessage, Fix: fixMessage, Doc: docURL}
+		errorMessage1 := fmt.Errorf("%s failed:\n%s ", strings.Join(cmd.Args, " "), out)
+		cmd := exec.CommandContext(ctx, path, "-NoProfile", "-NonInteractive", "@(Get-Wmiobject Win32_ComputerSystem).HypervisorPresent")
+		out, err := cmd.CombinedOutput()
+		if err != nil {
+			errorMessage2 := fmt.Errorf("%s failed:\n%s ", strings.Join(cmd.Args, " "), out)
+			errorMessage := fmt.Errorf("%s\n%s", errorMessage1, errorMessage2)
```

Comment:
Can we improve the variable names of the error messages, we have `errorMessage`, `errorMessage1`, and `errorMessage2`.  Add some context `cimError` or something like that